docs/modules/configuring/con-virtualkafkacluster-tcp.adoc

```diff
@@ -0,0 +1,62 @@
+// file included in the following:
+//
+// kroxylicious-operator/assemblies/assembly-operator-deploy-a-proxy.adoc
+
+[id='con-configuring-virtualkafkacluster-{context}']
+= A virtual cluster for on-cluster access without TLS
```

Comment:
Wondering where the right place to add a note describing our limitation around only allowing a single port identifies broker ingress within a KafkaProxy is. This module is sort-of in the right area, but I guess it's more a part of the ingress concepts where we might enumerate the different ingress types.